### PR TITLE
Add formatter and appease the lint gods

### PR DESCRIPTION
**Pull request by warmwaffles**
When I was compiling this the `name` in the `mix.exs` file is a character string instead of a binary. This resolves that warning. This is going to require a new version to be released to allow downstream consumers to stop receiving compilation warnings.

I've also added the formatter to the linter task and run it initially in this PR so that it passes.

The warning I receive when compiling this library without this change:

```
    warning: single-quoted strings represent charlists. Use ~c"" if you indeed want a charlist or use "" instead
    │
 10 │       name: 'ExImageInfo',
    │             ~
    │
    └─ /home/warmwaffles/src/github.com/******/*******/deps/ex_image_info/mix.exs:10:13
```

### CODE REVIEW

**Comment by rNoz** [comment]
Thanks, Matthew!

**Review by rNoz — APPROVED** [review]

**Comment by rNoz** [comment]
Feel free to bump version to 0.2.5 in this PR before merging, if you consider enough convenient to omit this warning to all library users. Otherwise, if you plan to extend some functionality "soon", we can do bump it then.
Your call ;)

**Comment by rNoz** [comment]
Would be great to include this as an auto formatter when developing https://github.com/adobe/elixir-styler
I have started to use it a few months ago and I like it. Nevertheless, we can do it in the future.

**Comment by warmwaffles** [comment]
I have not used `elixir-styler` yet, as the last time I looked at it, it was still a release candidate.

I'll bump the version number. But seeing how you have experience in setting the styler up, I would love to see how it was done.

File: mix.exs
Comment (warmwaffles) [inline mix.exs:10]: This was the main warning I was trying to fix.

**Comment by rNoz** [comment]
Sure, it was just a suggestion for the future. We can do it when it's stable.
I can push another PR with styler, and we can decide if merging that.

**Pull request merged**
